### DISCUSSION TITLE
Add a loader until the comments load

### DISCUSSION THREAD

**Discussion by orpheustelos7** [dailydotdev/daily#1141] [Feature Requests]
I was looking at the comments and I saw this

![image](https://github.com/dailydotdev/daily/assets/99677867/c37854f3-47f2-49d4-8e4e-298c34cde6d5)

The number of comments here are being shown as 5 but there are no comments, I was actually going to report a bug but luckily the comments loaded after a while, so I thought that it would be better if there was a loader or spinner displaying comments loading status.

**Comment by idoshamun** [comment]
Thanks for reporting @aneesh1024!
We'll take care of that :)

**Comment by idoshamun** [comment]
It's fixed now. Thank you!
